helm/docs/common-configuration.md

```diff
@@ -0,0 +1,450 @@
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.aks.fileShareName
+      </td>
+      <td>
+       The <i>File share name</i> of your File share instance.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.aks.azureStorageAccountName
+      </td>
+      <td>
+       The <i>base64 encoded storage account name</i> of the File share instance in your storage account.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.aks.azureStorageAccountKey
+      </td>
+      <td>
+       The <i>base64 encoded storage account key</i> of the File share instance in your storage account.
+      </td>
+    </tr>
+</table>
+<br/>
+
+> **NOTE:** The Azure storage account credentials will be maintained in a secret named `bold-azure-secret`
+
+4. On-Premise
+  
+```console
+clusterProvider: onpremise
+    
+persistentVolume:
+  onpremise:
+    hostPath: /run/desktop/mnt/host/<local_directory>
+```
+
+<br/>
+<table>
+    <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       clusterProvider
+      </td>
+      <td>
+       The type of kubernetes cluster provider you are using. In this case the clusterProvider value is <i>onpremise</i>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.onpremise.hostPath
+      </td>
+      <td>
+       The shared folder path in your host machine.<br/><br/>
+       Ex: Create a folder in your machine to store the shared folders for application usage.
+       <i>D://app/shared</i><br/><br/>
+       The corresponding value for the above shared folder will be,
+       <i>/run/desktop/mnt/host/d/app/shared</i>
+      </td>
+    </tr>
+</table>
+<br/>
+
+## Image
+
+We need to give a image tag for Bold BI, Bold Reports and ID. By default set a latest version of a image tags. You can change a tag based on requirments and the tag overides the image tag whose default in the chart appVersion.<br>
+```console
+image:
+  idRepo: gcr.io/boldbi-294612
+  biRepo: gcr.io/boldbi-294612
+  reportsRepo: gcr.io/boldreports
+  # Overrides the image tag whose default is the chart appVersion.
+  idTag: 5.2.48
+  biTag: 5.2.48
+  reportsTag: 4.2.52
+````
+Repository details are available in image section to refer the image tags. No need to change the repository details for idRepo, biRepo and reportsRepo.
+
+## Version
+
+By default latest version of Bold BI, Bold Reports and IDP in the version section. You can change a version based on requirements.
+```console
+versions:
+  idp: "3.1.21"
+  bi: "5.2.48"
+  reports: "4.2.52"
+````
+> **NOTE:**  We need give a latest version of `idp` when you have latest version of Bold BI or Bold Reports version
+
+## Load Balancing
+
+### Ingress-Nginx
+
+By default Nginx is used as reverse proxy for Bold BI. To configure load balancing in Bold BI take a look at the below snipet. 
```

Comment:
By default, Nginx is used as a reverse proxy for Bold BI. To configure load balancing in Bold BI take a look at the below snippet.